multiraft/multiraft.go

```diff
@@ -720,6 +722,7 @@ func (s *state) handleWriteResponse(response *writeResponse, readyGroups map[uin
 					NodeID:     NodeID(cc.NodeID),
 					ChangeType: cc.Type,
 					ConfState:  *cs,
+					Payload:    payload,
```

Comment:
I'm working on this part now: adding a callback to EventCommandCommitted so we don't call multiNode.ApplyConfChange until the application is ready (and we can abort the change on error). What's the best way to force the transaction to fail for testing?